robot_skills/src/robot_skills/util/transformations.py

```diff
@@ -128,6 +128,42 @@ def tf_transform(coordinates, inputframe, outputframe, tf_listener):
     output_coordinates = tf_listener.transformPoint(outputframe, ps, fixed_frame=inputframe)
     return output_coordinates.point
 
+
+def img_recognitions_to_rois(recognitions):
```

Comment:
This has nothing to do with `transformations.py`

Reply:
It wasn't sure where to put it. I thought this was a better place than adding it as a static method. What do you think?

Reply:
As said before: a separate module besides transformations.py, eg. image_operations.py